### DISCUSSION TITLE
Flux.Experimental namespace

### DISCUSSION THREAD

**Discussion by ToucheSir** [FluxML/ML-Coordination-Tracker#29] [Ideas]
Given all the back and forth we've had recently about the best time to land features and how to create a smooth deprecation path for them, I would like to get your thoughts on whether we should add such a namespace. Prior art includes Julia base as well as the 2 big Python DL frameworks.

**Comment by darsnack** [comment]
I think this would greatly improve our ability to iterate and release features. We're currently a pre-1.0 package that needs to act like a post-1.0 package. This inhibits our ability to push features out and get feedback.
